lib/sass-npm-importer.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var path = require('path');
+var findup = require('findup-sync');
+
+var cache = {};
+
+function sassNpmImporter(url, prev, done) {
+  // Fall back to old URL
```

Comment:
If you put `newUrl = url` in the except block, you could probably ditch the comment.